src/data/showcases.js

```diff
@@ -1208,6 +1208,15 @@ export const Showcases = [
     source: null,
     tags: ["analytics", "pooltool"],
   },
+  {
+    title: "Peer Review Money",
```

Comment:
```suggestion
    title: "Peer Review Money: ADA Economics",
```
I cannot suggest that the tool itself be retitled, but I think Peer Review Money is too vague/general as a portfolio title.  Note @katomm I am not violating everyone's favourite capitalisation rules for Ada because I believe this site would mainly be used for ADA (trading symbol) price predictions.

Reply:
Changing the name is not something that I would like to have. Is there anything that I can do for the approval? I just want the community to find the site on the list and use it, since I got some good feedback.

Reply:
The best we can do now is to invite some other reviewers, so I'll do that right now.